src/sage/matroids/chow_ring_ideal.py

```diff
@@ -137,32 +162,30 @@ class ChowRingIdeal_nonaug(ChowRingIdeal):
 
     Chow ring ideal of uniform matroid of rank 3 on 6 elements::
 
-        sage: ch = matroids.Uniform(3, 6).chow_ring(QQ, False)
+        sage: ch = matroids.Uniform(3, 6).chow_ring(QQ, False, 'fy')
         sage: ch.defining_ideal()
         Chow ring ideal of U(3, 6): Matroid of rank 3 on 6 elements with
-         circuit-closures {3: {{0, 1, 2, 3, 4, 5}}} - non augmented
-        sage: ch = matroids.catalog.Fano().chow_ring(QQ, False)
+        circuit-closures {3: {{0, 1, 2, 3, 4, 5}}} - non augmented in
+        Feitchner-Yuzvinksy presentation
```

Comment:
```suggestion
         circuit-closures {3: {{0, 1, 2, 3, 4, 5}}} - non augmented in
         Feitchner-Yuzvinksy presentation
```
I like these spaces to make it clear they are continuations of the previous line. Same for elsewhere in the files.